deps/picojson-1.3.0/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: cpp
```

Comment:
can we remove unnecessary files from the dependency?
travis configuration, gitmodules